kubeflow-roles/base/cluster-roles.yaml

```diff
@@ -0,0 +1,39 @@
+---
```

Comment:
Can we put this file as part of profile entry?
Since we will bind them through profile & kfam?

Reply:
> Can we put this file as part of profile entry?
> Since we will bind them through profile & kfam?

@kunmingg We have discussed this above in the comments:
1. https://github.com/kubeflow/manifests/pull/374#issuecomment-533793390
1. https://github.com/kubeflow/manifests/pull/374#issuecomment-533810673
1. https://github.com/kubeflow/manifests/pull/374#pullrequestreview-292117387

It looks like we have 2 votes for a separate folder and 2 for having this in profiles.